.github/dependabot.yml

```diff
@@ -28,5 +28,3 @@ updates:
     ignore:
       # protobuf must be handled manually
       - dependency-name: google.golang.org/protobuf
-      # Our own dependencies are handled during releases
-      - dependency-name: github.com/submariner-io/*
```

Comment:
Does this actually work on devel commits? What we want is bumps to the latest commit on the devel branches; I wonder if this won’t cause dependabot to submit PRs updating to the latest tag, either on devel or on the last release branch once it exists (which will do the wrong thing).

Reply:
I'll remove that change for now, send a different PR with a different attempt.